kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -525,23 +559,63 @@ def _do_get_lockspaces():
+        if not sanlock.io_failed(r.get_renewal_last_result()):
+            logger.warn("sanlock delta lease renewal failed with {} for vg {}, this doesn't look like an IO error".format(r.get_renewal_last_result(), vg))
+            res.no_way = True
+            return res
 
-        return victims
+        def is_heartbeat_timeout(timeout):
+            return abs(last_check - last_success) > timeout
+
+        try:
+            last_check = int(linux.monotime())
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)